### PR TITLE
Issue 729 update learning resources

### PR DESCRIPTION
**Pull request by lerongil**
update learning content and capitalize beginner and advanced

### CODE REVIEW

File: new-content/learning-resources/qiskit-blog.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 ---
-title: Qiskit blog
-to: https://medium.com/qiskit
-image: /images/landing-page/feature-hardware.jpg
-ctaLabel: Read the docs
+title: Qiskit Medium
+to: https://medium.com/qiskit/qiskit-and-its-fundamental-elements-bcd7ead80492
```
Comment (y4izus) [inline new-content/learning-resources/qiskit-blog.md:3]: This link point to a post related to qiskit elements. As far as I know, qiskit elements will disappear, so, what about pointing to the main qiskit page on medium?

File: new-content/learning-resources/textbook-1-3.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Qiskit Textbook Chapters 1 - 3
+to: https://qiskit.org/textbook
```
Comment (y4izus) [inline new-content/learning-resources/textbook-1-3.md:3]: This link points to the main page of textbook, not to the first chapter. We already have a link to textbook, makes sense to have two cards linking to the same page?

File: new-content/learning-resources/tutorials.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Tutorials
-to: https://github.com/Qiskit/qiskit-tutorials
-image: /images/landing-page/feature-hardware.jpg
-ctaLabel: Go to Khan Academy
+title: Qiskit Tutorial
+to: https://qiskit.org/documentation/tutorials/circuits/1_getting_started_with_qiskit.html
```
Comment (y4izus) [inline new-content/learning-resources/tutorials.md:3]: On the footer, tutorials point to `https://github.com/Qiskit/qiskit-tutorials` not to the main page of circuits

File: new-content/learning-resources/textbook-1-3.md (hunk 2 above)
Comment (lerongil) [reply new-content/learning-resources/textbook-1-3.md:3]: Good catch. Thank you!

File: new-content/learning-resources/qiskit-blog.md (hunk 1 above)
Comment (lerongil) [reply new-content/learning-resources/qiskit-blog.md:3]: The elements are still a part of the qiskit SDK so when you learn how to use Qiskit you will still run into them. In this case - it's ok to use them.

File: new-content/learning-resources/tutorials.md (hunk 3 above)
Comment (lerongil) [reply new-content/learning-resources/tutorials.md:3]: That is fine, this is pointing to the first page of the tutorials which is intended for users who select one day and advance.

**Pull request merged**